snakemake/exceptions.py

```
@@ -469,6 +468,14 @@ class SpawnedJobError(Exception):
     pass
 
 
+class CheckSumMismatchException(Exception):
```

Please inherit from WorkflowError. This way, Snakemake will by default hide the stack trace and just display the message.

---

Ah, good idea!